platform-ui-skin/src/main/webapp/skin/less/core/helpers.less

```
@@ -17,6 +17,12 @@
   Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+@font-face {
+  font-family: "ellipsis-font";
+  src: local('DejaVu Sans');
```

Please rename font family and define the default fonts as defined in `@fontFamily` variable in order to preserve the font family of displayed text inside activities.

```suggestion
  font-family: "Hide Line Clamp Font";
  src:
    local('Helvetica')
    local('arial')
    local('sans-serif');
```

---

fixed

---

fixed

---

fixed